guides/_subsections/archive-a-recording.md

```diff
@@ -41,14 +41,14 @@ any state (`RUNNING`, `STOPPED`, etc.), and may even be a snapshot.
       summary="Navigate to the <i>Archived Recordings</i> tab"
       image-name="2.3.0/archive-a-recording-2.png"
       caption="
-        Once the recording has been archived, a new entry will appear in the
-        target JVM's <i>Archived Recordings</i> table. All recordings that were
+        Once the <code>Recording</code> has been <code>Archived</code>, a new entry will appear in the
```

Comment:
arguably, "archived" in this case is a verb and is referring to the act of archiving a recording, not an *archived* recording. Thus, I'm unsure whether it should be in a code block....

Reply:
this is indeed just being nitpicky though

Reply:
yes it is a verb.

Reply:
the other archived below it is also a verb. :)